components/Navigation.js

```diff
@@ -30,6 +30,7 @@ export default function Navigation() {
   return (
     <NavbarContainer>
       <NavItem href="/">Home</NavItem>
+      <NavItem href="/create">New Card</NavItem>
       <NavItem href="/archive">Archive</NavItem>
```

Comment:
Right now, `NavItem` is a styled `<a>` tag, which causes a full page reload when navigating. In Next.js, we should always use `<Link>` for client-side navigation to ensure fast transitions.